docs/guides/token_guide.md

```diff
@@ -31,3 +27,124 @@ Below is an overview of how the migration will take place:
+
+#### Create a Migration Bundle
+Then, for the addresses we receive, we basically split them into chunks / bundles. We could create a single bundle for all addresses and migrate them, but we want to avoid that and cap the number of address a single bundle can contain, which happens [here](https://github.com/iotaledger/firefly/blob/develop/packages/shared/lib/migration.ts#L279-L300). Note that for spent addresses, we only have a single input per bundle.
+
+#### Check for spent addresses
+The next step is to check if there are spent addresses on a user's seed. If there are, we bundle mine those addresses. This is done through createMigrationBundle by setting the third param mine to true. For the rest of the bundles, we will keep this param to false. createMigrationBundle will return us the migration bundle.
+
+#### Send them to the new network
+The last step is to simply use sendMigrationBundle and broadcast them to the network.
+
+Another important point is how we select inputs. You can see how Firefly accomplishes this [here](https://github.com/iotaledger/firefly/blob/develop/packages/shared/lib/migration.ts#L228-L277).
+
+
+Basically, we need to make sure we create bundles that have accumulative balance >= 1Mi.
+
+#### Migration Code Example with Node.js
+This example creates a new database, account, and migrates funds from the legacy network to the chrysalis network.
+
+ 
+```javascript=
+require('dotenv').config()
+
+async function run() {
+    const { AccountManager, SignerType } = require('@iota/wallet')
+    const manager = new AccountManager({
+        storagePath: './migration-database',
+    })
+    manager.setStrongholdPassword(process.env.SH_PASSWORD)
+    manager.storeMnemonic(SignerType.Stronghold)
+
+    const account = await manager.createAccount({
+        clientOptions: { node: "https://api.lb-0.testnet.chrysalis2.com", localPow: true },
+        alias: 'Migration',
+    })
+
+    console.log('Account created:', account.alias())
+
+    const nodes = ['https://nodes.devnet.iota.org']
+    const seed = process.env.MIGRATION_SEED
+    const migrationData = await manager.getMigrationData(
+      nodes,
+      seed,
+      {
+        permanode: 'https://chronicle.iota.org/api'
+      }
+    )
+    console.log(migrationData)
+    const bundle = await manager.createMigrationBundle(seed, migrationData.inputs.map(input => input.index), {
+      logFileName: 'iota-migration.log'
+    })
+    await manager.sendMigrationBundle(nodes, bundle.bundleHash)
+}
+
+run()
+```
+If you are new to [wallet.rs](https://github.com/iotaledger/wallet.rs), please check out our [Wallet Library Documentation](https://chrysalis.docs.iota.org/libraries/wallet.html).
```

Comment:
Maybe add an additional note fore the backup, because all you get from the example is the stronghold file